src/main/java/com/google/sps/JobModel.java

```diff
@@ -129,4 +129,20 @@ private void getMetrics(Dataflow dataflowService) throws IOException, IllegalArg
         }
       }    
     }
+
+    public static JobModel createJob(String projectId, Job job, Dataflow dataflowService)
+        throws IOException, IllegalArgumentException {
+            String state = state = job.getCurrentState();
```

Comment:
```suggestion
            String state = job.getCurrentState();
```

Should this just be this?

Reply:
Yes, you are right. Thanks!